---?image=assets/images/background_dna.jpg
<div class="right white">
<strong>My Life as a Bioinformatician</strong><br/><br/>

Saskia Hiltemann<br/>
May 2018, LIACS
</div>

---

### About Me

+++

### About Me

- Physics & Astronomy
- Bachelor Computer Science
- Master Bioinformatics
- PhD Bioinformatics at Erasmus Medical Center

Note:
- Always liked puzzles
- Drew me to science
- Did programmeermethoded in physics/astronomy and fell in love
- Bioinformatics is greatest puzzle of all

+++

### PhD Thesis

![](assets/images/academia/thesis.png)

---

### Overview

+++

### Overview

- What is Bioinformatics?
- Academica vs Industry
- Genomic Sequencing 101
- My Projects
 - Pipeline building with Galaxy
 - Cancer Biomarker Discovery
 - Microbiome Antibiotic Resistance

+++

- Don't save questions til the end, just ask!

---
### What is bioinformatics?

+++

### What is bioinformatics?

*The use of computer science, mathematics, and information theory to model and analyze biological systems, especially systems involving genetic material.*

![](assets/images/bioinformatics/venn.png)

+++

### Interdisciplinary

![scale-40](assets/images/bioinformatics/venn2.png)

+++

### Why Bioinformatics

*“There’s millions and millions of unsolved problems. Biology is so digital, and incredibly complicated, but incredibly useful. [..] Biology easily has 500 years of exciting problems to work on, it’s at that level." - Donald Knuth*


![scale-60](assets/images/cartoons/dna-code.png)



Note:
Famous computer scientist Donald Knuth has discussed his concerns that computer science in the future will be “pretty much working on refinements of well-explored things”, whereas “Biology easily has 500 years of exciting problems to work on”.

And this seems like an understatement, as we are discovering more and more complexity

Full Quote: “There’s millions and millions of unsolved problems [in Biology]. Biology is so digital, and incredibly complicated, but incredibly useful…It is hard for me to say confidently that, after fifty more years of explosive growth of computer science, there will still be a lot of fascinating unsolved problems at peoples’ fingertips, that it won’t be pretty much working on refinements of well-explored things. Maybe all of the simple stuff and the really great stuff has been discovered. It may not be true, but I can’t predict an unending growth. I can’t be as confident about computer science as I can about biology. Biology easily has 500 years of exciting problems to work on, it’s at that level."


+++

### What is bioinformatics

<div class="left small">
<ul>
<li> **Modeling.** simulations, and writing equations to represent biological systems. Gain understanding of the biology.</li><br>
<li> **Data analysis.** Going from raw data, to clean data, to statistical and visual interpretations of the results. </li><br>
<li> **Software development.** Developing software to do bioinformatics analysis, big enough software products to publish as independent methods papers and to be used by other scientists.</li><br>
</ul>
</div>

<div class="right">
![scale-90](assets/images/bioinformatics/research-trinity.png)
</div>

Note:

- Very broad definition
- Often used interchangably with *computational biology*
- **Computational biology** = the study of biology using computational techniques.  The goal is to learn new biology, knowledge about living sytems. It is about science.
- **Bioinformatics** = the creation of tools (algorithms, databases) that solve problems. The goal is to build useful tools that work on biological data. It is about engineering.
- In practice: most people do a bit of both.
- Different definitions based on who you ask
- *Systems biology*, *quantitative biology*

https://diamondagedatascience.com/2017/07/05/bioinformatics-vs-computational-biology/

https://rbaltman.wordpress.com/2009/02/18/bioinformatics-computational-biology-same-no/

### What is bioinformatics?

Many different areas of research
- Healthcare
- Ecology
- Agriculture
- Evolutionary Development
- Image processing (e.g. MRI)
- ..

+++

### What is bioinformatics?

Many different areas of research
- **Healthcare (Me)**
- Ecology
- Agriculture
- Evolutionary Development
- Image processing (e.g. MRI)
- ..

---
### ErasmusMC

+++

### ErasmusMC

- Research Hospital
- Bioinformatics Department
- Support all other departments in their research
- Support outside labs


![scale-50](assets/images/erasmusmc.jpg)

+++

### The Biologist and the Informatician

- Work closely together on any project
- Domain knowledge vs computational knowledge

![scale-50](assets/images/cartoons/experimental-vs-computational.jpeg)

+++

### Biologists

<div class="left70">
<ul>
<li>Not typically trained in programming</li>
<li>Excel is their analysis tool of choice </li>
<li>No longer possible in big data era</li>
</ul>
</div>

<div class="right30">
![](assets/images/cartoons/excel.png)
</div>

<div class="bottom">
![border scale-90](assets/images/bioinformatics/bioinfblackbox.jpg)
</div>


+++

### What I do

<div class="left70">
<ul>
<li> Jack(ie) of all trades </li>
<li> Research, software development, pipeline building, system administration, IT infrastructure, training </li>
<li> Define and adopt best practices, make it durable, maintainable</li>
<li> Use many concepts from BSc/MSc </li>
  <ul><li> Algorithms, databases, image analysis, pattern recognition, machine learning, ..</li></ul>
</ul>
</div>

<div class="right30">
![](assets/images/bioinformatics/jackofalltrades.jpg)
</div>

Note:
- gotta like a good puzzle
- problem solving
- troubleshooting

write papers
go to conferences
teach workshops
develop pipelines
data analysis

+++

### Health Care

- Working with human data: data security important


---
### Academia vs Industry

Note:
source: https://www.slideshare.net/hafidztio/academia-vs-industry

+++

### Salary

- You will earn more at a company
- Academia offers other advantages: flexibility, freedom

<div class="bottom">
![scale-60](assets/images/academia/academicsalaries.gif)
</div>

Note:
- Industry pays more, but academia has lot of pros too
- Disclaimer: I am biased and never worked in industry


+++

### Flexibility

<div class="left">
<br><br>
**Academia**
<br>
<ul>
<li> Informal, flat hierarchy </li>
<li> Flexible schedule </li>
</ul>
</div>
<div class="right">
<br><br>
**Industry**
<br>
<ul>
<li>More structured</li>
<li>Core working hours</li>
</ul>
</div>
<br><br><br><br><br><br>

<div class="bottom">
![scale-80](assets/images/academia/academicdresscode.gif)
</div>

Note:
- I don't have to ask for time off, just take it
- As long as projects stay on track, can make your own hours
- Treated as equal from the start, ideas valued
- YMMV! (and other countries can be very different)

+++

### Flexibility

- But: this flexibililty breeds a lot of workaholics
- Take care to keep a healthy work-life balance!

<br>

<div class="bottom">
![scale-60](assets/images/academia/bioinformaticiancanwork.gif)
</div>

Note:
- Make sure to guard your work-life balance

+++

### Flexibility

- But: this flexibililty breeds a lot of workaholics
- Take care to keep a healthy work-life balance!

<br>

<div class="bottom">
![scale-60](assets/images/academia/academicsaturdays.gif)
</div>

Note:
- Make sure to guard your work-life balance



+++

### Freedom

<div class="left">
<br><br>
**Academia**
<br>
<ul>
<li> Freedom to choose projects </li>
<li> Lot of travel </li>
</ul>
</div>

<div class="right">
<br><br>
**Industry**
<br>
<ul>
<li>Constrained by needs of company</li>
</ul>
</div>
<br><br><br><br><br><br>


<div class="bottom">
![scale-60](assets/images/academia/academicfreedom.gif)
</div>

Note:
- Great freedom to choose projects
- Within limitation of course
- YMMV as always

+++

### Outputs


<div class="left">
<br><br>
**Academia**
<br>
<ul>
<li> Obtain new knowledge </li>
<li> Publish papers, obtain funding </li>
<li> Train new scientists </li>
</ul>
<br><br>
![scale-60](assets/images/academia/journal-article.jpg)

</div>

<div class="right">
<br><br>
**Industry**
<br>
<ul>
<li> Earn money for the company</li>
<li> Software, drugs, patents, services </li>
</ul>
<br><br>

![scale-50](assets/images/academia/pill-head.jpg)

</div>

Note:

- Academia does basic research
- Industry converts knowledge into real-world applications

+++

### Academia **AND** Industry

Often close collaborations between academia and companies

<div class="bottom">
![scale-60](assets/images/academia/collaboration.png)
</div>

Note:
- Especially in larger consortia
- Sometimes a requirement for grant proposals
- Obtain the knowledge and convert it to usable products in one project

+++

### Imposter Syndrome

- Feeling of being a fraud
- "Everybody else knows more than I do"
- Own successes dismissed as luck, not deserved

<div class-"bottom">
![scale-60](assets/images/academia/impostor-syndrome.jpg)
<div>

Note:
- very common
- esp in such a fast evolving field, both CS and bioinf, can never know everything
- remind yourself that this is common
- don't let it stop you

Impostor syndrome is a psychological pattern in which people doubt their accomplishments and have a persistent, often internalized fear of being exposed as a "fraud". The term was coined in 1978 by clinical psychologists Pauline R. Clance and Suzanne A. Imes. Despite external evidence of their competence, those exhibiting the syndrome remain convinced that they are frauds and do not deserve the success they have achieved.

Proof of success is dismissed as luck, timing, or as a result of deceiving others into thinking they are more intelligent and competent than they believe themselves to be. While early research focused on the prevalence among high-achieving women, impostor syndrome has been found to affect both men and women, in roughly equal numbers.


---

### Genome Sequencing

+++

### DNA

- DNA responsible for all the hereditary informatin in an organism.
- Natural variation in genetic code between individuals.
- Two copies of each chromosome and gene (maternal and paternal)
- Some mutations can disrupt cell functions and cause disease.

<div class="left">
![scale-80](assets/images/bioinformatics/genome-project.png)
</div>
<div class="right">
![scale-50](assets/images/bioinformatics/music-of-life-dna.jpg)
</div>
+++

### Central dogma

![scale-50](assets/images/bioinformatics/dna.png)

+++

### From genes to phenotype

- Genes: DNA translated to RNA, RNA to protein
- Proteins perform cell functions, control expression of other genes, etc
- Disease often result of mutations in gene sequence/expression

![scale-90](assets/images/bioinformatics/whatisbioinf.png)

+++

### Human Genome Project

<div class="left70">
<ul>
<li> Began in 1990 </li>
<li> Goal: determine sequence of all 3.3 billion nucleotides and map locations of all genes </li>
<li> 20 labs in 6 countries </li>
<li> Completed in 2003 </li>
<li> Cost: 3 billion </li>
<br>
<li> Revolutionized the field </li>
</ul>
</div>

<div class="right30">
![](assets/images/cartoons/human_genome.jpg)
</div>

+++

### 15 years later..

- Cost for sequencing: < $1000
- USB sequencers

![scale-50](assets/images/bioinformatics/minion-usb-sequencer.jpg)


+++

### Sequence Data Explosion

- Cost of sequencing decreasing exponentially
- Cost of computing decreasing
- Number of samples sequenced increasing exponentially

![scale-70](assets/images/bioinformatics/sequencing_explosion.jpg)

Note:
data explosion, moores law on steroids


+++

### Genome Sequencing

- Can only reliably determine sequence for small fragments of DNA

![scale-70](assets/images/bioinformatics/sequence-reads.png)

+++

### Genome Sequencing

- Fragment must be mapped back to reference genome
- Not all organisms have a reference genome (de novo assembly)

![scale-70](assets/images/bioinformatics/shotgun_sequencing.jpg)

Note:
- String matching algorithm
  - allow for mutations
  - imperfections
  - often framed as graph problem
  - or machine learning/ natural computing algorithms

- Faster algorithms continually wanted to keep up with data explosion


+++

### Genome Sequencing

<div class="left small">
<ul>
<li> Like a jigsaw puzzle, except:</li>
 <ul>
 <li> Billions pieces</li>
 <li> Missing pieces (coverage bias)</li>
 <li> Broken pieces (sequencing error)</li>
 <li> Duplicate pieces (repeats)</li>
 <li> Pieces from different puzzle (contamination)</li>
 </ul><br>
<li> Non-human samples:</li>
 <ul>
 <li> Do not have the lid (de novo)</li>
 <li> No corner/edge pieces (bacteria, circular genome)</li>
 </ul>
</ul>
</div>

<div class="right">
![](assets/images/cartoons/genomepuzzle.gif)
</div>

Note:

### Genome Sequencing

- Base calling (usually done by software integrated in sequencing machines)
- Mapping and variant calling
 - How is this individual different from the reference genome?
- Annotation, data integration and visualisation
 - Can we predict effect of mutation?
 - Compare to other patients
- Aggregation of heterogeneous datasets into a coherent reference source

Note:
Four main analysis tiers
- want to know where sample is different from reference
- want to compare this to other samples/knowledge databases

+++

### Mapping and Variant Calling

![scale-60](assets/images/bioinformatics/mappedreads.png)

SNV = Single Nucleotide Variation (1 basepair mutation)


Note:
- we are interested to see where a person's DNA is different
- 2 copies of each gene so would expect either 0%, 50% or 100% mutation
- 3% is likely just sequencing error

+++

### Genome Sequencing

![scale-60](assets/images/bioinformatics/mappedreads2.png)

- **Possibilities for observed difference:**
 - True mutation
 - Error generated in library prep (in the lab)
 - Base calling error
 - Misalignment (mapping error)
 - Error in reference genome

Note:
- Of course previous slide was the ideal case, and in reality not always so straightforward

+++

### Genome Sequencing

![](assets/images/cartoons/statistics.gif)

+++

### Genomic Variations

<div class="left">
<ul>
<li>Different types of variants</li>
 <ul>
 <li> SNVs</li>
 <li> Insertions</li>
 <li> Deletions</li>
 <li> Inversions</li>
 <li> Substitutions</li>
 <li> Transocations</li>
 </ul><br>
<li> Both within and between chromosomes </li>
</div>

<div class="right">
![](assets/images/cancer/svs.jpg)
</div>

+++

### Cancer

<div class="left">
<br><br><br><br>
<ul>
<li> Cancer is a genetic disease</li><br>
<li> Germline (hereditary) 5%-10%</li><br>
<li> Somatic (non-hereditary)</li>
   <ul>
  <li> Diet/tobacco/alcohol</li>
  <li> Radiation/toxins</li>
  <li> Bacteria/viruses</li>
  </ul>
</ul>
</div>

<div class="right">
![scale-70](assets/images/cancer/cells.jpg)
![scale-70](assets/images/cancer/causes.jpg)
</div>

+++

### Cancer

<div class="left small">
<ul>
<li> Every time a cell divides, DNA is copied (imperfectly)</li><br>
<li> If mutations occur in important genes, more mutations can accumulate</li>
 <ul>
 <li> Disrupt DNA repair genes</li>
 <li> Activate Cell proliferation genes</li>
 <li> Evade programmed cell death</li>
 </ul><br>
<li> Late stage cancers can have accrued hundreds of thousands of mutations</li>
</ul>
</div>
<div class="right">
![](assets/images/cancer/hallmarks.png)
</div>

Note:
- multipe puzzles, similar but different, don't know how many
- structural variations
- fusion genes
- chromothripsis

+++

### Cancer: Fusion Genes


+++

### Cancer: Chromothripsis


+++

### Cancer: Our research

- Biomarker discovery
- Improve diagnosis/prognosis
- Predict drug response


---
### Pipelines and Automation


+++

### Pipelines and Automation

- Don't reinvent the wheel
- Combine existing tools into pipelines
- Make it easy for researchers/biologistis/clinicians!

![scale-60](assets/images/cartoons/redbutton.gif)

+++

### Data Standards

- Field evolving so fast, there often aren't established file formats
- Challenge for building pipelines

![](assets/images/cartoons/standards.png)

+++

### Bioinformatics tools

<div class="left">
<ul>
<li> Linux </li>
<li> Command Line </li>
<li> Open source </li>
 <div class="transparant">
 <br>
 <li> Not always top quality </li>
 <li> ..or maintained well </li>
 <li> Can be hard to install </li>
 <li> ..and poorly documented <li>
 </div>
</ul>
</div>

<div class="right">
![](assets/images/microbiota/mothur_commandline.png)
</div>

+++

### Bioinformatics tools

<div class="left">
<ul>
<li> Linux </li>
<li> Command Line </li>
<li> Open source </li>
<br>
<li> Not always top quality </li>
<li> ..or maintained well </li>
<li> Can be hard to install </li>
<li> ..and poorly documented </li>
</ul>
</div>

<div class="right">
![](assets/images/microbiota/mothur_commandline.png)
</div>

Note:
- command line is very powerful, but not very user friendly
- especially not for non-programmers
- do not want to run everybody's analysis for them -> automate it!


+++

### Galaxy

- Provide web interface for command line tools
- Tool shed: app store
- Graphical workflow builder

+++

### Galaxy



---
### Cancer


---
### Example Project: Human Microbiome Analysis

+++

### Microbiota

![Video](https://www.youtube.com/embed/htbeJhtFAXw)

+++

### A Brief History

+++

### Microscopy

<div class="left small center">

<ul>
  <li> Antonie van Leeuwenhoek is commonly known as the *Father of Microbiology* </li>

![scale-30](assets/images/microbiota/leeuwenhoek.jpg)

  <li> In 1676, he discovered **animalcules** when he turned his microscope to a drop of rain water </li>
</ul>

</div>

<div class="right">

<!-- ![scale-70](assets/images/microbiota/animalcules.gif) -->
![](assets/images/microbiota/leeuwenhoek-microscope-animalcules.jpg)

---
</div>

Note:
Antonie van Leeuwenhoek was Dutch scientist, best known for his work in microscopy

In 1676 he turned his microscope at a drop of rain water and was surprised to find
what looked like tiny animals, which he called **animalcules**

+++

### Spontaneous Generation

<div class="small">
<ul>
<li>Spallanzani (1729–1799), Pasteur (1822–1895)</li>
<li>Boiling a broth kills microbes, and they will not return unless exposed to air</li>
</ul>
</div>

<div>
![scale-50](assets/images/microbiota/pasteur-experiment.jpeg)
</div>

Note:

+++

### Germ Theory

<div class="small">
<ul>
<li>Robert Koch (1843–1910)</li>
<li>Demonstrated a causal link between microbes and disease</li>
</ul>
</div>

<div>
![scale-60](assets/images/microbiota/koch-postulates.jpeg)
---
</div>

+++

### Why study the microbiome?

<div class="block">
<div class="left small middle">
<br><br>
<ul>
  <li>Humans are full of micro-organisms</li>
  <ul>
   <li> Skin, gut, oral cavity, lungs, eyes, ..</li>
   <li> Impacting health, drug efficacy </li>
  </ul>
  <br>
  <li> > 10 times more cells than you </li>
  <li> > 100 times more genes than you </li>
  <li> > 1000 different species </li>
</ul>
</div>
<div class="right">

![](assets/images/microbiota/human_microbiome.png)

</div>
</div>

Sometimes referred to as your **second genome**

Note:

- many applications in ecology
  - soil analysis -> agriculture

- as a hospital, we are interested in humans
- humans are also full of microorganisms
  - e.g. skin, gut, oral and nasal cavities, lungs, even eyes
  - impact on health and drug efficacy


+++


### Dysbiosis

<div class="block">
<div class="left small leftalign">

<ul>
<li> Microbial imbalance in the body </li>
<ul><li>Symbionts, Commensals, Pathobionts</li></ul>
<br>

<li> Causes </li>
 <ul>
  <li> Infection  </li>
  <li> Lifestyle (diet, stress) </li>
  <li> Host genetics </li>
  <li> Medications </li>
 </ul><br>

<li> Linked to a large number of diseases </li>
 <ul>
  <li> Obesity, Diabetes </li>
  <li> Asthma, Allergies </li>
  <li> Crohn's disease, Ulcerative colitis </li>
  <li> Cancer, IBS, Autism </li>
 </ul>

</div>

<div class="right tiny">
![unbalanced scale-50](assets/images/microbiota/dysbiosis-balance.png)
![diseases](assets/images/microbiota/microbiota-disease.jpg)

</div>
</div>

<div class="tiny">
doi:10.1038/nri2515
</div>

Note:

Dysbiosis is a general term for an imbalance in your body's microbiome

A healthy microbiota contains a balanced composition
of many classes of bacteria.

**Symbionts** are organisms with known health-
promoting functions.

**Commensals** are permanent residents of this complex ecosystem and provide no benefit
or detriment to the host (at least to our knowledge).

**Pathobionts** are also permanent residents of the microbiota
and have the potential to induce pathology.

with **dysbiosis** there is an unnatural shift in the composition of
the microbiota, which results in either a reduction in the
numbers of symbionts and/or an increase in the numbers
of pathobionts.

The causes for this are not entirely clear,
but are likely to include recent societal advances in
developed countries. The result is non-specific inflammation,
which may predispose certain genetically susceptible
people to inflammatory disease and may be caused by
pathogens, which are opportunistic organisms that cause
acute inflammation.

+++

### Dysbiosis

Can be caused by improper use of antibiotics

<div class="block">
<div class="three-left small">
![healthy forest](assets/images/microbiota/forest-healthy.jpg)
Healthy microbiome
</div>
<div class="three-middle small">
![after antibiotics](assets/images/microbiota/forest-burnt.jpg)
After antibiotics
</div>
<div class="three-right small">
![monoculture](assets/images/microbiota/forest-monoculture.jpg)
Monoculture
</div>
</div>


Note:
There are many causes of dysbiosis, but one of thm is improper use of antibiotics

Imaging a healthy microbiome as a lush, green rainforest

When you administer antibiototics it is as if you burn this forest to the ground

But this can lead to a situation where a single species can


+++


### Streeklab Haarlem

<div class="left small alignleft">

<br><br>
<ul>
<li> Microbial Diagnostics Laboratory</li><br>
<li> Services large number of </li>
 <ul>
  <li> Hospitals </li>
  <li> Clinics </li>
  <li> GPs </li>
 </ul><br>
<li>Samples (e.g. blood, stool, urine) are sent to the lab and analysed</li>
 <ul>
   <li> Determine microbiota present </li>
   <li> Determine antibiotic resistance </li>
   <li> Advise a treatment </li>
 </ul>
</ul>

</div>

<div class="right center">

![](assets/images/microbiota/slh-logo2.png)
![](assets/images/microbiota/slh-envelope.png)

</div>

+++

### Streeklab Haarlem

<div class="left small">
<ul>
 <li>Samples are cultured</li><br>
 <li>Species determined by </li>
  <ul>
   <li> MALDI-TOF MS</li>
   <li> Visual inspection</li>
  </ul><br>
 <li>Limitations</li>
  <ul>
   <li> Medium determines what grows</li>
   <li> Some species difficult to culture</li>
  </ul><br>
 <li>Pilot study: MYcrobiota</li>
   <ul>
   <li> 16S rRNA sequencing</li>
   <li> Can this augment current practices?</li>
  </ul>
</ul>

</div>
<div class="right">
![](assets/images/microbiota/slh-culture.jpg)
</div>

Note:
- MALDI: Matrix Assisted Laser Desorption/Ionization
- TOF MS: Time of Flight Mass spectrometry

- Bacteria grown on selective medium
 - Not all bacteria thrive equally (or at all) on medium
 - optimized to quickly show known pathogens
 - but "rest" can also be interesting
 - anaerobic bacteria are hard to grow in lab

- 16S cannot show antibiotic resistance, and only reliable down to genus level
- Time: 16S usually equally quick
 - 16S 2-3 days
 - cultures: mostly 24-48 hours, some types difficult to culture and can take 2 weeks

- Costs: NGS higher, but when culture is negative, cheaper than hospitalization



+++

### Sequencing Approaches

- Shotgun sequencing
- Amplicon sequencing

![scale-70](assets/images/microbiota/miseq.png)

+++

### Shotgun Sequencing

<div class="left small">
<ul>
<li> Sequence everything </li><br>
<li> Expensive ($1000+ per sample) </li><br>
<li> More information
 <ul>
  <li> Functional prediction</li>
  <li> Antibiotic resistance </li>
  <li> Unknown organisms </li>
</ul><br>
<li> Higher complexity </li>
</ul>
</div>

<div class="right">
![](assets/images/microbiota/shotgun-puzzles.png)
</div>

+++

### Amplicon Sequencing

<div class="left small">
<br>
<ul>
<li> Targetted approach </li>
  <ul>
   <li> 16S/18S rRNA gene </li>
   <li> Present in all bacteria and archaea </li>
  </ul></br>
<li>No contamination from: </li>
  <ul>
   <li>Host</li>
   <li>Environmental fungi, plants</li>
 </ul>
</ul>
![scale-50](assets/images/microbiota/16S-gene-ecoli.png)
</div>


<div class="right">
![](assets/images/microbiota/amplicon-puzzles2.png)
</div>

Note:
- where shotgun was looking at whole puzzle, this is more like
  looking at only topleft corner of each puzzle

+++

### 16S rRNA gene

<div class="left small">
<ul>
 <!--<li> Ribosomal RNA gene </li>
 <ul>
   <li>~1500 bp long</li>
 </ul><br>-->
 <li> Highly conserved regions</li>
  <ul>
   <li>easy to target across all bacteria</li>
  </ul><br>
<li>Variable regions</li>
 <ul>
  <li>V1, V2,.., V9 </li>
  <li>distinguish between different bacteria</li>
 </ul>
</ul>
<br>
![](assets/images/microbiota/16S_variableregions.jpg)
</div>

<div class="right">
![scale-70](assets/images/microbiota/16S-structure.jpg)
</div>


Note:
- 16S gene is around 1500 nucleotides long
- Strong secondary structure, folds into 3D structure
- S stands for Svedberg (unit of time, time it takes to sediment in centrifuge)

+++

### Amplicon Sequencing

<div class="left small">
<br>
<ul>
<li>Pros:</li>
 <ul>
  <li>Cheap: ($50-$100 per sample)</li>
  <li>Well-established</li>
  <li>Lot of reference databases</li>
 </ul> <br>
<li>Cons: </li>
  <ul>
  <li>Choice of V-region can bias results</li>
  <li>Difficult to resolve species/strains</li>
  </ul>
</ul>
</div>

<div class="right tiny">
![](assets/images/microbiota/bias-vregions.png)
doi:10.1038/nature11209
</div>

Note:
Image: number of OTUs identified by genus for V13 region (y-axis)
and V35 region (x-axis)

For example for Coprococcus, 65 OTUs identified when using V35 region,
3 OTUs with V13 region.

TODO: move bias image til after explanation of OTUs?

+++

### Streeklab Haarlem


![scale-70](assets/images/microbiota/16S_end-to-end.png)
![scale-50](assets/images/microbiota/so_much_data.png)

+++

### Galaxy!

![](assets/images/microbiota/workflow.png)

+++

### Pipeline

![](assets/images/microbiota/analysis_pipeline.png)


+++

### Pre-processing

![scale-70](assets/images/microbiota/preprocessing.png)

- many ways to filter and trim your data
- tradeoff between quality and amount of information retained

+++

### Chimera removal


<div class="left small">
<br><br>
<ul>
<li>Greek mythology </li>
 <ul>
  <li>Hybrid animals</li>
 </ul><br>
 <li>Sequencing data </li>
 <ul>
  <li> Hybrid sequences </li>
  <li> Created during PCR</li>
 </ul><br>
 <li> Correction Methods </li>
 <ul>
  <li>In Silico</li>
  <li>Micelle PCR</li>
 </ul>
</ul>
</div>

<div class="right">
![scale-40](assets/images/microbiota/chimera-animal.jpg)
<br>
![scale-80](assets/images/microbiota/chimeras.png)
</div>

+++

### OTU Clustering

**OTU**: Operational Taxonomic Unit

<div class="left">
![](assets/images/microbiota/otu-clusters.png)
</div>
<div class="right">
![](assets/images/microbiota/otu.png)
</div>

Note:
- clusters of closely related sequences
- proxy for species/genus (97%=species, but perhaps too close to sequencing error territory to be reliable)
- control for sequencing/PCR errors
- for amplicon data, this is not done so much in shotgun
-

+++

### OTU Clustering

<div class="left small">
<ul>
 <li>Clustering Approaches:</li>
 <ul>
  <li>De novo</li>
 </ul>
</ul>
</div>

<div class="right">
![](assets/images/microbiota/otu-clusters.png)
![](assets/images/microbiota/otu-clustering-reference-based-placeholder.png)
</div>

Note:
OTU definition (taxonomy, sequence similarity, phylogeny)
clustering strategy (de novo, open reference, closed reference)
distance metrics (pairwise alignment, multiple alignment, alignment-free)
algorithm (hierarchical clustering, seed-based, model-based)

+++

### OTU Clustering

<div class="left small">
<ul>
 <li>Clustering Approaches:</li>
 <ul>
  <li>De novo</li>
  <li>Closed reference</li>
 </ul>
</ul>
</div>

<div class="right">
![](assets/images/microbiota/otu-clusters.png)
![](assets/images/microbiota/otu-clustering-reference-based.png)
</div>

Note:
OTU definition (taxonomy, sequence similarity, phylogeny)
clustering strategy (de novo, open reference, closed reference)
distance metrics (pairwise alignment, multiple alignment, alignment-free)
algorithm (hierarchical clustering, seed-based, model-based)


+++

### OTU Clustering

<div class="left small">
<ul>
 <li>Clustering Approaches:</li>
 <ul>
  <li>De novo</li>
  <li>Closed reference</li>
  <li>Open Reference</li>
 </ul>
</ul>
</div>

<div class="right">
![](assets/images/microbiota/otu-clusters.png)
![](assets/images/microbiota/otu-clustering-reference-based.png)
</div>

Note:
OTU definition (taxonomy, sequence similarity, phylogeny)
clustering strategy (de novo, open reference, closed reference)
distance metrics (pairwise alignment, multiple alignment, alignment-free)
algorithm (hierarchical clustering, seed-based, model-based)

+++


### OTU Clustering

<div class="left small">
<ul>
 <li>Clustering Approaches:</li>
 <ul>
  <li>De novo</li>
  <li>Closed reference</li>
  <li>Open reference</li>
 </ul>
 <br>
 <li>Distance Metrics</li>
 <ul>
  <li>Pairwise alignment</li>
  <li>Multiple alignment</li>
  <li>Alignment-free</li>
 </ul>
 <br>
 <li>Algorithms</li>
 <ul>
  <li>Hierarchical clustering</li>
  <li>Seed-based clustering</li>
  <li>Model-based clustering</li>
 </ul>
</ul>
</div>

<div class="right">
![](assets/images/microbiota/otu-clusters.png)
![](assets/images/microbiota/otu-clustering-reference-based.png)
</div>

Note:
OTU definition (taxonomy, sequence similarity, phylogeny)
clustering strategy (de novo, open reference, closed reference)
distance metrics (pairwise alignment, multiple alignment, alignment-free)
algorithm (hierarchical clustering, seed-based, model-based)

+++

### Assigning Taxonomy

<div class="block">
<div class="left small">
<ul>
 <li>Methods</li>
 <ul>
  <li>Sequence alignment</li>
  <li>K-mer based methods</li>
  <li>Tree based methods</li>
  <li>Machine learning</li>
 </ul>
 <br>
 <li>Reference Databases</li>
 <ul>
  <li>SILVA</li>
  <li>Green Genes</li>
  <li>RDP</li>
  <li>NCBI</li>
 </ul>
</ul>
</div>

<div class="right">
![scale-70](assets/images/microbiota/taxonomic-classification.jpeg)
![](assets/images/microbiota/silva-greengenes.png)
</div>
</div>

<div class="tiny">
Comparison: [10.1186/s12864-017-3501-4]((https://doi.org/10.1186/s12864-017-3501-4), [10.1093/nar/gkn491](https://doi.org/10.1093/nar/gkn491)
</div>

Note:
- often done after OTU clustering, but not necessarily
- GreenGenes not updated since 2013

+++

### Assigning taxonomy

![](assets/images/microbiota/taxonomy-methods.jpeg)

<div class="tiny">
doi: 10.1093/nar/gkn491
</div>

Note:

- composition at phylym level
- each box a different dataset
- each subpanel (row) a different clasisfication method
- x-axis: region sequenced
- y-axis: abundance as fraction of total
- conclusion: choice of classifier has great impact on species identified

figure legend:
Compositions at the phylum level for each of the three datasets: (a) Guerrero Negro mat, (b) Human gut and (c) Mouse gut, using a range of different methods (separate subpanels within each group). The x-axis of each graph shows region sequenced. The y-axis shows abundance as a fraction of the total number of sequences in the community. The legend shows colors for phyla (consistent across graphs).

+++

### OTU table

![](assets/images/microbiota/otutable.png)

Note:
domain, kingdom, phylum, class, order, family, genus, species

+++

### Diversity metrics

- Alpha Diversity
- Beta Diversity

Note:
Alpha diversity: diversity within 1 sample
Beta diversity: diversity between samples

in our case: every patient considered on its own, so only alpha diversity
is considered.

+++

### Alpha Diversity

Diversity *within* a sample

![scale-90](assets/images/microbiota/alpha-diversity.png)


+++

### Alpha Diversity


<div class="small left70">
<ul>
 <li>Simple approach: count number of OTUs</li>
 <br>
 <li>Doesnt take into account relatedness of OTUs</li>
  <ul><li>Phylogenetic distance metrics</li></ul>
 <br>
 <li>Doesnt take into account relative distribution</li>
 <ul>
  <li>Sample A: 3 species at 98%,1%,1%</li>
  <li>Sample B: 3 species at 33%,33%,33%</li>
 </ul>
</ul>
</div>

<div class="right30">
![](assets/images/microbiota/phylogenetic-distance.png)
</div>

Note:
- PD: sum of branch lengths
- in our pipeline we report several diversity metrics


+++

### Diversity

<div class="block small">
*"As many diversity measures as there are ambitious ecologists"*
<br><br>
shannon, chao, pd, ace, simpson, sobs, jack, npshannon, smithwilson, heip
bergerparker, boney, efron, shen, solow, bootstrap, qstat, coverage, anderberg,
hamming, jclass, jest, ochiai, canberra, thetayc, invsimpson, ...
</div>


<div class="small">
https://www.mothur.org/wiki/Calculators
</div>

+++

### Rarefaction

Have I captured the full diversity of my sample?

![](assets/images/microbiota/rarefaction.gif)

Calculate diversity for randomly selected subset of sequences in each sample


+++

### Post-processing

<div class="left small">
<ul>
<li>3 replicates</li>
 <ul><li>OTU must be present in all of them</li></ul>
 <br>
 <li>Negative Extraction Control Sample</li>
  <ul><li>subtract background sample</li></ul>
 <br>
 <li>Internal Control Sample</li>
  <ul><li>relative abundance -> copies</li></ul>

</ul>
</div>

<div class="right">

![](assets/images/microbiota/negative-control.jpg)
</div>

Note:
- tetramix: equimolar mix of 4 species
- 2500 to 2.5 16S rRNA gene copies per bacterial species

Synthetic microbial community (SMC) samples tested comprised equimolar 16S rRNA gene copies derived from C. perfringens, S. aureus, H. influenzae, and M. catarrhalis and ranging from 2,500 to 2.5 16S rRNA gene copies per bacterial species. Averages of triplicate micPCR/NGS and triplicate traditional PCR/NGS results are shown in 100% stacked bars before and after correction for contaminating bacterial DNA. The correction of contaminating bacterial DNA comprises two steps: 1) eliminating OTUs that could not be reproducibly measured in triplicate experiments, and 2) subtracting 16S rRNA gene copies that were also quantified in triplicate measurements of a negative extraction control (NEC) sample. Values within bars represent the calculated number of 16S rRNA gene copies per bacterial OTU.i


+++

### Visualisations

+++

### Krona

![scale-50](assets/images/microbiota/krona.png)

<div class="tiny">
doi: 10.1186/1471-2105-12-385
</div>
---

<embed src="https://marbl.github.io/Krona/examples/rdp.krona.html">

+++

### Phinch

![scale-70](assets/images/microbiota/phinch_overviewpage.png)

<div class="tiny">
doi: 10.1101/009944
</div>

Note:
- useful for multi-sample analyses


+++

### iReport

![](assets/images/microbiota/ireport-mycrobiota.png)

+++

### Results: Post-operative wounds

<div class="left">
![](assets/images/microbiota/cultures.png)
</div>


<div class="right">
![](assets/images/microbiota/miseq2.png)
</div>

<div class="tiny">
<table>
<tr><td>Taxonomy</td><td>CFU</td><td>%</td>   <td>Taxonomy</td><td>Copies</td><td>%</td></tr>
<tr><td>Skin flora</td><td>+</td><td>33%</td>  <td>Anaerobic bacteria (12)</td><td>251.692</td><td>87%</td> </tr>
<tr><td>Staphylococcus aureus</td><td>++</td><td>67%</td>   <td>Streptococcus</td><td>30.408</td><td>10%</td></tr>
<tr><td></td>                     <td></td>   <td></td>     <td>Staphylococcus aureus</td><td>8.960</td><td>3%</td></tr>
</table>

</div>

<br>

<div class="small">
<ul>
 <li>Identify and quantify anaerobic bacteria just as easily as aerobic bacteria</li>
</ul>
</div>

Note:

CFU: Colony Forming Unit

+++

### Results: Septic Arthritis

<div class="left70 small">
<ul>
 <li>19 patients</li>
 <li>Suspected septic arthritis</li>
 <li>Culture negative</li>

 <br>
 <li>MYcrobiota:</li>
 <ul>
  <li>confirmed 12/19 negative cultures</li>
  <li>detected bacteria in 7/19 negative cultures</li>
  <ul>
   <li>Parvimonasi, Prevotella, Ruminococcus</li>
   <li>Kingella -> confirmed with Kingella kingae specific PCR</li>
   <li>Ureaplasma </li>
   <li>Enterococus, Turicella</li>
  </ul>
 </ul>


</ul>
</div>

<div class="right30">
![](assets/images/microbiota/arthritis.png)
</div>


+++

### Results: Septic Arthritis


<div class="left70 small">
<ul>
<li>Suspected Septic Arthritis patient</li>
<li>History of dental extraction</li>
<li>Culture negative</li>
<br>

<li>MYcrobiota: Prevotella denticola</li>
 <ul>
  <li>70K copies</li>
  <li>typical inhabitant of the oral flora</li>
  <li>Hard to culture under standard conditions</li>
 </ul>
</ul>
</div>

<div class="right30">
![](assets/images/microbiota/arthritis.png)
</div>


+++

### Results: Septic Arthritis 2

<div class="left70 small">
<ul>
<li>Arthritis patient</li>
<li>History: operation of the intestine</li>
<li>Culture negative</li>
<br>

<li>MYcrobiota: Rhuminococcus species</li>
  <ul>
  <li>192K copies</li>
  <li>Inhabitant of the intestine</li>
  <li>Hard to culture under standard conditions</li>
  </ul>
</ul>
</div>

<div class="right30">
![](assets/images/microbiota/arthritis.png)
</div>

+++

### Results: Septic Arthritis 3

<div class="left70 small">
<ul>
<li>Arthritis patient</li>
<li>Culture Negative</li>
<br>
<li>MYcrobiota: Neisseria meningitidis</li>
 <ul>
  <li>895K copies</li>
  <li>inhabitant of the throat</li>
  <li>causative agent of bacterial meningitis</li>
  <li>probably not culturable due to use of antibiotics</li>
 </ul>

</ul>
</div>

<div class="right30">
![](assets/images/microbiota/arthritis.png)
</div>

+++

### Results: Septic Arthritis 4

<div class="left70 small">
<ul>
<li>Arthritis patient</li>
<li>Culture Negative</li>
<br>
<li>Ureaplasma</li>
 <ul>
  <li>1200 copies</li>
  <li>not culturable in routine diagnostics</li>
 </ul>
</ul>
</div>

<div class="right30">
![](assets/images/microbiota/arthritis.png)
</div>



+++

### Results: Hip Transplant

<div class="left70 small">
<ul>
<li>Infected hip implant</li>
<li> 5 samples from different locations</li>
<li> No culture data available</li>
<br>
<li> Result: Staphylococcus </li>
  <ul>
  <li> 1.3K, 53K, 10K, 742K, 1072K gene copies</li>
  <li> most likely contaminations during operation</li>
  </ul>
</ul>
</div>
<div class="right30">
![](assets/images/microbiota/hipimplant.jpg)
</div>

+++

### Conclusion

<div class="small">
<ul>
 <li>MYcrobiota offers very precise quantitative microbiota analyses</li>
 <li>Can provide valuable insight in cases where cultures were negative</li>
 <li>Well suited for samples with low biomass</li>
 <li>Clinicians can use it without bioinformatics expertise needed</li>
</ul>
</div>
---

### Take-home messages

- Bioinformatics is a very diverse field
- Continually changing field: keep learning
- Do not need to know biology, will pick it up along the way
- Job security! ..big shortage of good bioinformaticians

- Want to do an internship? Contact us!

@fa[code-fork]

---

### Acknowledgements

<div class="left tiny">
<br>
<ul>
 <li>**ErasmusMC Bioinformatics**</li>
 <ul>
  <li>Andrew Stubbs</li>
  <li>Saskia Hiltemann</li>
  <li>Youri Hoogstrate</li>
  <li>David van Zessen</li>
  <li>Yunlei Li</li>
  <li>Bas Horsman</li>
  <li>Rick Jansen</li>
 </ul>
<br>
<li>**ErasmusMC Urology**</li>
 <ul>
  <li>Guido Jenster</li>
  <li>Ines Teles-Alves</li>
  <li>Rene Bottcher</li>
 </ul>
 <br>
<li>**ErasmusMC Clinical Microbiology**</li>
 <ul>
  <li>John Hays</li>
  <li>Stefan Boers</li>
 </ul>
</ul>
</div>
<div class="right tiny">
<br>
<ul>
<li>**Streeklab Haarlem**</li>
 <ul>
  <li>Ruud Jansen</li>
  <li>Linda van de Nes-Reijnen</li>
  <li>Wil van der Reijden</li>
  <li>Elana Meijer</li>
 </ul>
<br>
<li>**Web Comics**</li>
<ul>
<li>XKCD</li>
<li>PhD Comics</li>
<li>BioComicals</li>
<li>TwistedDoodles</li>
<li>RedPen/BlackPen</li>
</ul>
</ul>
<br>
![scale-50](assets/images/logo-emc.png)
</div>


---

### Questions?

![](assets/images/questions.png)

